CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Version 2.8.0
```

Comment:
Should this be updated to 2.9.0?

Reply:
Whoops. 🤦 

Thanks for pointing this out!

Reply:
Thanks!